libraries/helpers.rb

```diff
@@ -307,5 +307,25 @@ def package_options
 
       options
     end
+
+    #
+    # Checks the deprecated properties of chef-ingredient and prints warning
+    # messages if any of them are being used.
+    #
+    def check_deprecated_properties
```

Comment:
I updated the above function to include `chef-server-ha-provisioning`. I think we'll need a separate if check for the old `push-client` and `push-server` naming. Good times.